batch/src/main/scala/uk/gov/ons/addressindex/utils/SqlHelper.scala

```diff
@@ -309,28 +309,42 @@ object SqlHelper {
 
         val countryCode = if (outputNisra.headOption.nonEmpty) "N" else outputLpis.headOption.flatMap(_.get("country").map(_.toString)).getOrElse("E")
 
+        val EnglishNag = outputLpis.find(_.getOrElse("language", "ENG") == "ENG")
```

Comment:
probably want to follow casing convention with englishNag rather than EnglishNag

Reply:
yup, that's true. Maybe I thought English should always be capitalised :-)